src/main/java/minegame159/meteorclient/systems/commands/PrefixListener.java

```diff
@@ -0,0 +1,28 @@
+package minegame159.meteorclient.systems.commands;
+
+import meteordevelopment.orbit.EventHandler;
+import minegame159.meteorclient.MeteorClient;
+import minegame159.meteorclient.events.meteor.CharTypedEvent;
+import minegame159.meteorclient.systems.config.Config;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.gui.screen.ChatScreen;
+
+public class PrefixListener {
+    private static final MinecraftClient mc = MinecraftClient.getInstance();
+    private static final Config config = Config.get();
+
+    public static void init() {
+        MeteorClient.EVENT_BUS.subscribe(PrefixListener.class);
+    }
+
+    @EventHandler
+    private static void onCharTyped(CharTypedEvent event) {
+        if (mc.currentScreen != null) return;
+        if (!config.openChatOnPrefix) return;
+
+        if (event.c == config.prefix.charAt(0)) {
```

Comment:
What if someone has a prefix that is multiple characters. This is the reason why i never implemented this because it just doesn't make sense if you have a prefix thats more than a single character

Reply:
If the prefix is longer then it shouldn't check for the key and do nothing.

Reply:
I made it toggleable, so if someone doesn't want it they can disable it